src/components/BindingModal.tsx

```diff
@@ -1,8 +1,6 @@
 import React from "react";
 import Modal from "react-modal";
-
```

Comment:
i would be inclined to keep current import style (external libraries, blank line, outside folder imports, blank line, same folder imports)

Reply:
We've been inconsistent about this but I've pushed a commit https://github.com/cjquines/qboard/pull/46/commits/f4d2067809e77b071a707d35ec147f38cc16ee8b addressing this.